ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/WebSocketTest.kt

```diff
@@ -89,6 +95,54 @@ class WebSocketTest : ClientLoader() {
         }
     }
 
+    @Test
+    fun testWsHandshakeHeaders() {
+        if(PlatformUtils.IS_ BROWSER) return // browser websocket client does not support custom headers so the test get's ignored
```

Comment:
With this space, you code does not compile

Reply:
OOOPS I didn't see that